classes/author.rb

```diff
@@ -0,0 +1,17 @@
+require_relative './item'
+class Author
```

Comment:
Please apply some of the changes in the comment below this one on this class as well.

Reply:
Thank you @Timbar09  for your comment, I will take them into consideration, kindly also check the Item class to ensure that the publish_date argument is set well.

Reply:
That's great @aubinleyoung. I have updated the argument in my branch which I have merged into the `dev` branch.